### PR TITLE
Use aws_cloudtrail event source plugin to showcase Ansible Validated Content

### PR DESCRIPTION
**Pull request by alinabuzachis**
Use aws_cloudtrail event source plugin to showcase Ansible Validated Content

### CODE REVIEW

**Review by jillr — COMMENTED** [review]
One question, everything else lgtm

File: rulebooks/aws_manage_cloudtrail_encryption.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+- name: Rules for cloud.aws_ops to ensure the cloudtrail is encrypted
+  hosts: all
+
+  sources:
+    - ansible.eda.aws_cloudtrail:
+        region_name: 'us-east-1'
```
[the remark this answers is not in the document]
Comment (alinabuzachis) [reply rulebooks/aws_manage_cloudtrail_encryption.yml]: I replaced with vars.region_name instead

File: playbooks/eda/aws_restore_cloudtrail.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  tasks:
+  - block:
+    - name: Include 'cloud.aws_ops.awsconfig_multiregion_cloudtrail' role
+      ansible.builtin.include_role:
+        name: cloud.aws_ops.awsconfig_multiregion_cloudtrail
+      vars:
+        operation: create
+        bucket_name: "{{ s3_bucket_name }}"
+        key_prefix: "{{ key_prefix }}"
+        trail_name: "{{ cloudtrail_name }}"
```
[the remark this answers is not in the document]
Comment (alinabuzachis) [reply playbooks/eda/aws_restore_cloudtrail.yml:13]: They are coming from the vars.yml file. I added the usage guide here https://github.com/redhat-cop/cloud.aws_ops/pull/32. If you think it's better I can also add the vars file, but an example is shown in the docs.

[the remark this answers is not in the document]
Comment (gravesm) [reply playbooks/eda/aws_restore_cloudtrail.yml:13]: Yeah, #32 is fine.

**Review by gravesm — APPROVED** [review]

**Review by jillr — APPROVED** [review]

**Pull request merged**